lib/ret_web/templates/page/manifest.webmanifest.eex

```diff
@@ -0,0 +1,40 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "<%= @app_name %>",
+  "description": "<%= @app_description %>",
+  "short_name": "<%= @app_name %>",
+  "icons": [
+    {
+      "src": "<%= @app_icon %>",
+      "sizes": "120x120",
+      "type": "image/png"
+    },
+    {
+      "src": "<%= @app_icon %>",
+      "sizes": "180x180",
+      "type": "image/png"
+    },
+    {
+      "src": "<%= @app_icon %>",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "<%= @app_icon %>",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "scope": "/",
+  "start_url": "/",
+  "display": "standalone",
+  "orientation": "portrait",
+  "theme_color": "#000",
```

Comment:
we can eventually get smarter here, but i don't think we should worry about dark vs light just yet (i expect us to have more themes)